stories/topics/proc-aap-aws-application-upgrade.adoc

```diff
@@ -0,0 +1,164 @@
+[id="proc-aap-aws-application-upgrade"]
+
+You can upgrade your existing {AAPonAWS} deployment to the newer version. The upgrade process covers upgrading of Automation Hub and Automation Controller. The upgrade process takes roughly the same amount of time as a {AAPonAWS} deployment install. Follow the steps below to ensure a smooth upgrade process.
+
+[NOTE]
+=====
+{AAPonAWS} supports sequential upgrades. All upgrades should be no more than one minor version behind what you are currently upgrading to. For example, in order to upgrade to {AAPonAWS} to 2.2.20230215, you must be on the first version of {AAPonAWS}.
+=====
+
+[NOTE]
+=====
+The upgrade process does not support
+=====
+
+== Requirements
+
+This upgrade process requires the following:
+- Linux or macOS system (where the Upgrader will be run)
+- Docker
```

Comment:
You require (can this be expanded) why do you need Docker?

Reply:
At least so far, we didn't have the bandwidth to test with other container engines. Since we tested on macOS, Docker was the chosen one.

@zkayyali812 ^